packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -97,6 +121,19 @@ export default function (server: Server, ctx: AppContext) {
             .execute()
         }
 
+        if (
+          ctx.cfg.phoneVerification.required &&
+          input.body.verificationPhone
+        ) {
+          await dbTxn.db
+            .insertInto('phone_verification')
+            .values({
+              did,
+              phoneNumber: input.body.verificationPhone,
```

Comment:
Not sure if twilio spits out a normalized version of phone numbers—but if they do it might be worth tracking it here.  It's a real pain to validate and normalize phone numbers, and we'll end-up storing a wide variety of different formats in here.

Reply:
yup looks like it does 👍